sway-lib-std/src/u256.sw

```diff
@@ -241,17 +241,11 @@ impl U256 {
 
 impl core::ops::Ord for U256 {
     fn gt(self, other: Self) -> bool {
-        self.a > other.a ||
-                (self.a == other.a && (self.b > other.b ||
-                                      (self.b == other.b && (self.c > other.c ||
-                                                            (self.c == other.c && self.d > other.d)))) )
-        }
+        self.a > other.a || (self.a == other.a && (self.b > other.b || (self.b == other.b && (self.c > other.c || (self.c == other.c && self.d > other.d)))))
+    }
```

Comment:
chained booleans should have a newline and indent similarly to `if else` statements, this could also be another issue to open

Reply:
Agreed. Let's open an issue, but I don't think this should block us from enabling the CI checks.